.circleci/README.md

```diff
@@ -0,0 +1,56 @@
+# CircleCI
```

Comment:
Thanks for documenting this!

Reply:
Actually, could this mostly replace the ecosystem site doc? Lots of per-job detail that that seems to go away now

Reply:
Thanks, Les! I'll update that next. Maybe just to point here